From 2bdcfdb5d028fa711a6c4cb38a3d679faede739d Mon Sep 17 00:00:00 2001
From: Hannu Lyytinen <hannu.lyytinen@unikie.com>
Date: Tue, 21 Feb 2023 15:52:10 +0200
Subject: [PATCH] Add seL4 tracebuffer reserved mem type

Upstream-Status: Inappropriate [seL4-specific]

Signed-off-by: Hannu Lyytinen <hannu.lyytinen@unikie.com>
---
 drivers/of/platform.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/of/platform.c b/drivers/of/platform.c
--- a/drivers/of/platform.c
+++ b/drivers/of/platform.c
@@ -540,5 +540,6 @@ static const struct of_device_id reserved_mem_matches[] = {
 	{ .compatible = "ramoops" },
 	{ .compatible = "nvmem-rmem" },
 	{ .compatible = "google,open-dice" },
+	{ .compatible = "sel4_tracebuffer" },
 	{}
 };
